at_secondary/at_secondary_server/lib/src/server/at_secondary_impl.dart

```diff
@@ -153,6 +153,25 @@ class AtSecondaryServerImpl implements AtSecondaryServer {
     await commitLogCompactionJobInstance
         .scheduleCompactionJob(atCommitLogCompactionConfig);
 
+    if (AtSecondaryConfig.testingMode!) {
```

Comment:
Move all of these code blocks to separate methods ex: addConfigChangeListeners(); 

addConfigChangeListeners() {
     addCompactionConfigChangeListener();
     .. others
}

It is not good to have more than 50 to 80 lines in a single method for readability